projects/plugins/crm/src/automation/class-automation-workflow.php

```diff
@@ -111,14 +143,19 @@ class Automation_Workflow {
 	 * @param array $workflow_data The workflow data to be constructed.
 	 */
 	public function __construct( array $workflow_data ) {
-		$this->id              = $workflow_data['id'] ?? null;
-		$this->triggers        = $workflow_data['triggers'] ?? array();
-		$this->steps           = $workflow_data['steps'] ?? array();
-		$this->initial_step_id = $workflow_data['initial_step'] ?? null;
-		$this->name            = $workflow_data['name'];
-		$this->description     = $workflow_data['description'] ?? '';
-		$this->category        = $workflow_data['category'] ?? '';
-		$this->active          = $workflow_data['is_active'] ?? true;
+		$this->id           = $workflow_data['id'] ?? null;
+		$this->zbs_site     = $workflow_data['zbs_site'] ?? -1;
+		$this->zbs_owner    = $workflow_data['zbs_owner'] ?? -1;
+		$this->triggers     = $workflow_data['triggers'] ?? array();
+		$this->steps        = $workflow_data['steps'] ?? array();
+		$this->initial_step = $workflow_data['initial_step'] ?? '';
+		$this->name         = $workflow_data['name'];
+		$this->description  = $workflow_data['description'] ?? '';
+		$this->category     = $workflow_data['category'] ?? '';
+		$this->active       = $workflow_data['active'] ?? false;
+		$this->version      = $workflow_data['version'] ?? 1;
+		$this->created_at   = $workflow_data['created_at'] ?? 0;
+		$this->updated_at   = $workflow_data['updated_at'] ?? null;
```

Comment:
Is this still relevant with the workflow database changes?

Would it need to be like the following?

```suggestion
		$this->created_at   = $workflow_data['created_at'] ?? null;
		$this->updated_at   = $workflow_data['updated_at'] ?? null;
```

Reply:
It doesn't affect, but better with `null` for consistencies.